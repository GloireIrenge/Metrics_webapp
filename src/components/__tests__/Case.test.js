import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../../redux/configureStore';

import Case from '../Case';

const coun = {
  country: {
    abbreviation: 'CU',
    id: 'Cuba',
    life_expectancy: '76.2',
  },
};
it('<Case /> fectch country cases', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Provider store={store}>
        <Case country={coun} />
      </Provider>
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
