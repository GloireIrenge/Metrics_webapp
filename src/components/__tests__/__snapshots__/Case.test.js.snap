// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Case /> fectch country cases 1`] = `
<li
  className="Container odd"
>
  <a
    href="/country/undefined"
    onClick={[Function]}
  >
    <button
      onClick={[Function]}
      type="button"
    >
      <i
        className="fa-solid fa-circle-chevron-right fa-2x"
      />
    </button>
  </a>
  <div>
    <div>
      <p />
      <span />
    </div>
    <p>
      cases:
       
    </p>
  </div>
</li>
`;
